lib/node_modules/@stdlib/utils/compact-adjacency-matrix/test/test.js

```diff
@@ -20,31 +20,94 @@
 
 // MODULES //
 
-var tape = require( 'tape' );
-var instanceOf = require( '@stdlib/assert/instance-of' );
-var CompactAdjacencyMatrix = require( './../lib' );
+var tape = require('tape');
+var instanceOf = require('@stdlib/assert/instance-of');
+var CompactAdjacencyMatrix = require('./../lib');
 
 
 // TESTS //
 
-tape( 'main export is a function', function test( t ) {
-	t.ok( true, __filename );
-	t.strictEqual( typeof CompactAdjacencyMatrix, 'function', 'main export is a function' );
+tape('main export is a function', function test(t) {
+	t.ok(true, __filename);
+	t.strictEqual(typeof CompactAdjacencyMatrix, 'function', 'main export is a function');
 	t.end();
 });
 
-tape( 'the function is a constructor', function test( t ) {
-	var mat = new CompactAdjacencyMatrix( 4 );
-	t.strictEqual( instanceOf( mat, CompactAdjacencyMatrix ), true, 'returns an instance' );
+tape('the function is a constructor', function test(t) {
+	var mat = new CompactAdjacencyMatrix(4);
+	t.strictEqual(instanceOf(mat, CompactAdjacencyMatrix), true, 'returns an instance');
 	t.end();
 });
 
-tape( 'the constructor does not require the `new` keyword', function test( t ) {
+tape('the constructor does not require the `new` keyword', function test(t) {
 	var compactAdjacencyMatrix;
 	var mat;
 
 	compactAdjacencyMatrix = CompactAdjacencyMatrix;
-	mat = compactAdjacencyMatrix( 4 );
-	t.strictEqual( instanceOf( mat, CompactAdjacencyMatrix ), true, 'returns an instance' );
+	mat = compactAdjacencyMatrix(4);
+	t.strictEqual(instanceOf(mat, CompactAdjacencyMatrix), true, 'returns an instance');
 	t.end();
 });
+
+// adjacency Matrix can be depicted in form of undirected graph representation,directed graph represnetation and weighted graph representation
+
+tape('initialize the matrix correctly', function test(t) {
+	var mat = new CompactAdjacencyMatrix(4);
+	t.deepEqual(mat.matrix, [
```

Comment:
@uchiha-vivek Did you run these tests?

Based on what you have here, it does not appear so. There is no `matrix` attribute, and these tests are not written correctly.